tests/integration/RequestBuilder/ResetDataRequestBuilderTest.php

```diff
@@ -0,0 +1,29 @@
+<?php declare(strict_types=1);
+
+namespace Lmc\Matej\IntegrationTests\RequestBuilder;
+
+use Lmc\Matej\IntegrationTests\IntegrationTestCase;
+use Lmc\Matej\Model\Response\PlainResponse;
+
+/**
+ * @covers \Lmc\Matej\Model\Response\PlainResponse
+ * @covers \Lmc\Matej\RequestBuilder\ResetDataRequestBuilder
+ */
+class ResetDataRequestBuilderTest extends IntegrationTestCase
+{
+    /** @test */
+    public function shouldResetMatejData(): void
+    {
+        $response = static::createMatejInstance()
+            ->request()
+            ->resetData()
+            ->send();
+
+        $this->assertResponseCommandStatuses($response, 'OK');
+        $this->assertInstanceOf(PlainResponse::class, $response);
```

Comment:
As we no longer support PHP 5 client and this is guarded by the language itself, this could be omitted. (I will remove it elsewhere in separate PR.)

Reply:
ok, thanks

Reply:
removed

Reply:
Oh, I meant only this specific line with `assertInstanceOf` 😱. Rest of the test should be kept as it was :)

Reply:
sorry for misunderstanding, UT are back